dashboard/app/entities.go

```diff
@@ -222,6 +222,7 @@ type BugDiscussionInfo struct {
 type DiscussionSummary struct {
 	AllMessages      int
 	ExternalMessages int
+	FirstMessage     time.Time
```

Comment:
Isn't first discussion message always the reporting time?
We receive own report back and record it, no?
How do we want to use this DiscussionSummary.FirstMessage?

Reply:
> Isn't first discussion message always the reporting time?

If the discussion is too long, we don't store all message IDs. So the first stored message is not necessarily the first one.

> We receive own report back and record it, no?

What do you mean?

> How do we want to use this DiscussionSummary.FirstMessage?

I was planning to use it to determine the value for the `Link: ` field, but I've just noticed that I anyway also need to know the author of the first message, so it's probably better to change how messages are cropped - i.e. always keep the first message of the thread.